README.md

```diff
@@ -55,7 +55,7 @@ Once you have [installed Anaconda](https://docs.anaconda.com/anaconda/install/)
 This will give you a conda environment named `mlbook`, ready to use! Just activate it and you will have everything setup
```

Comment:
Would be nice to replace the name in the text too. 
```
This will give you a conda environment named `tf1`, ready to use! Just activate it and you will have everything setup
```